internal/providers/github/common.go

```diff
@@ -86,6 +88,7 @@ func NewGitHub(
 		client:   client,
 		cache:    cache,
 		delegate: delegate,
+		ghcrwrap: ghcr.FromGitHubClient(client, delegate.GetOwner()),
```

Comment:
Instead of this wrap, wouldn't it be better to implement the GHCR provider as embedded interface in the github provider (maybe through the app/oauth2 delegate)? Or is the intent to allow using the GHCR provider independently of github?

Reply:
that was the intent, that you could spawn a GHCR provider without needing the full blown github provider.